go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/chihaya/chihaya
+
+require (
+	github.com/SermoDigital/jose v0.0.0-20180104203859-803625baeddc
+	github.com/alicebob/gopher-json v0.0.0-20180125190556-5a6b3ba71ee6 // indirect
+	github.com/alicebob/miniredis v2.4.6+incompatible
+	github.com/anacrolix/torrent v1.0.0
+	github.com/go-redsync/redsync v1.1.1
+	github.com/gomodule/redigo v2.0.0+incompatible
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/julienschmidt/httprouter v1.2.0
+	github.com/mendsley/gojwk v0.0.0-20141217222730-4d5ec6e58103
+	github.com/minio/sha256-simd v0.0.0-20190131020904-2d45a736cd16
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.2
+	github.com/sirupsen/logrus v1.3.0
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/stretchr/testify v1.3.0
+	github.com/yuin/gopher-lua v0.0.0-20190206043414-8bfc7677f583 // indirect
+	gopkg.in/yaml.v2 v2.2.2
```

Comment:
You replaced the `redsync` dependency (which was also on gopkg.in) with the original from github, should we do the same for this one or not?

Reply:
While I don't like using `gopkg.in` these days, I don't think we should change this one. I believe this package has a magic import comment that says you have to load it from this URL. I had to move the other dependency to GitHub because it wasn't able to to be resolved for some strange reason.